src/components/Confirm/Previews.js

```diff
@@ -124,7 +124,12 @@ const Previews = localised(
               }}
             />
           ) : (
-            <PageTitle title={title} smaller={true} className={style.title} />
+            <PageTitle
+              title={title}
+              smaller={true}
```

Comment:
Should this be
```suggestion
              smaller
```
only